dialplan/asterisk/extensions_lib_context.conf

```diff
@@ -12,6 +12,7 @@ same  = n(return),Return
 ; ARG1 = caller user ID
 ; ARG2 = destination user ID, if applicable (optional)
 exten = s,1,NoOp(Counting calls for ${ARG1}@WAZO_USER)
+same  =   n,GotoIf($[${ARG1}]?:end)
```

Comment:
Shouldn't we still check for a destination user ID even if we're missing the caller user ID?

Reply:
No, because in the case we expect it to not be present it must be `-1`

Reply:
I see :+1: